build/charts/theia/templates/grafana/deployment.yaml

```diff
@@ -20,6 +20,14 @@ spec:
         fsGroup: 472
         supplementalGroups:
           - 0
+      initContainers:
+        - name: init-pv
+          image: projects.registry.vmware.com/antrea/busybox
+          imagePullPolicy: IfNotPresent
+          command: ["chown", "-R", "472:472", "/data"]
```

Comment:
A very pedant nit: if you have a chance, consider using the same style for the command both here in flow-visiiblity.yaml

Reply:
This part is rendered by Helm. I have not found a way to change the style here.